lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -112,10 +133,30 @@ def self.available_options
           FastlaneCore::ConfigItem.new(
             key: :skip_if_exists,
             description: 'If the file already exists in the S3 bucket, skip the upload (and report it in the logs), instead of failing with `user_error!`',
+            deprecated: 'Use if_exists instead',
+            verify_block: proc do
+              UI.important('The :skip_if_exists option is deprecated. Please use :if_exists instead.')
+            end,
```

Comment:
Turns out this was being overly cautious. Fastlane already prints a warning when using the deprecated option from within a lane:

<img width="1026" alt="image" src="https://github.com/wordpress-mobile/release-toolkit/assets/1218433/47e4006a-ec63-4beb-966f-81ab1b14b985">